src/coreclr/jit/ifconversion.cpp

```diff
@@ -286,8 +295,9 @@ bool OptIfConversionDsc::IfConvertCheckStmts(BasicBlock* fromBlock, IfConvertOpe
                 }
 
                 case GT_RETURN:
+                case GT_SWIFT_ERROR_RET:
```

Comment:
I think if-conversion supports `GT_RETURN` to transform
```
JTRUE(EQ(x, 0)) -> BB01, BB02

BB01:
RETURN(0)

BB02:
RETURN(1)
```

into

```
RETURN(SELECT(EQ(x, 0), 0, 1))
```
I don't see us being able to support this transform for `GT_SWIFT_ERROR_RET`, so I think we can't add this support here.